capi/src/visitors/links.ts

```diff
@@ -26,19 +25,26 @@ const getRoute = (
   transformerProps: t.TransformerProps,
 ): string => {
   const parsedURL = parse(url);
-  const {hash, query} = parsedURL;
+  const {hash} = parsedURL;
   let {path: urlPath} = parsedURL;
 
   if (urlPath) {
-    if (urlPath.includes("?")) {
-      urlPath = urlPath.split("?").shift() as string;
+    if (urlPath.includes("#")) {
+      urlPath = urlPath.split("#").shift() as string;
+    }
+    let query = "";
+    if (urlPath.includes("/q/")) {
```

Comment:
We need a check to make sure there is no page (`.md` file) named `q`

Reply:
Done